core/data/clients/InfluxEvent.go

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright 2017 Dell Inc.
```

Comment:
Are you from Dell?

Reply:
No. Sorry. It's my mistake.
I'll use SPDX headers as you say